app/uk.gov.hmrc.helptosavestridefrontend/connectors/HelpToSaveConnector.scala

```diff
@@ -178,12 +178,14 @@ class HelpToSaveConnectorImpl @Inject() (http:                              WSHt
         case _ ⇒
           logger.warn(s"createAccount returned a status: ${response.status} " +
             s"with response body: ${maskNino(response.body)}", nSIUserInfo.nino)
+          pagerDutyAlerting.alert("Received unexpected http status from the back end when calling the create account url")
```

Comment:
you have not changed the tests so they should fail now ? :)
Also don't forget to add these alerts to `alert-config` service too